_config.yml

```
@@ -81,14 +81,14 @@ google_analytics: G-WLMHGLDGZ8
\ No newline at end of file
+exclude:
+  - .sass-cache/
+  - .jekyll-cache/
+  - gemfiles/
+  - Gemfile
+  - Gemfile.lock
+  - node_modules/
+  - vendor/bundle/
+  - vendor/cache/
+  - vendor/gems/
+  - vendor/ruby/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)